spec/models/protected_food_drink_name_spec.rb

```diff
@@ -16,4 +16,12 @@
       expect(document.base_path).to eq("/protected-food-drink-names/sterea-ellada")
```

Comment:
Am I right in assuming that the Stringex approach generates the same slug as the manual Greek transliteration approach?

Reply:
Yeah, that's correct 👍